benchmark/infrastructure/BenchmarkToJson.cpp

```diff
@@ -72,9 +72,10 @@ nlohmann::json zipBenchmarkClassAndBenchmarkResultsToJson(
         benchmarkClassAndBenchmarkResults) {
   return transformIntoJsonArray(
       benchmarkClassAndBenchmarkResults, [](const auto& pair) {
-        return nlohmann::json{{"name", pair.first->name()},
-                              {"general metadata", pair.first->getMetadata()},
-                              {"measurements", pair.second}};
+        return nlohmann::json{
```

Comment:
Consider using `ordered_json` to preserve the insertion order.

Reply:
[c73f79f](https://github.com/ad-freiburg/qlever/pull/1221/commits/c73f79f3768742d0ab4dfaf97890b91c8721de9a)